cmd/kava/cmd/rocksdb/compact.go

```diff
@@ -0,0 +1,170 @@
+//go:build rocksdb
+// +build rocksdb
+
+package rocksdb
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/linxGnu/grocksdb"
+	"github.com/spf13/cobra"
+
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+var CompactRocksDBCmd = &cobra.Command{
+	Use:   "compact <state|blockstore>",
```

Comment:
These are good db's, but the main one we want to target for compaction is the application.db